doc/connector.md

```diff
@@ -1,46 +1,3 @@
-# Using this starter kit for a connector
+# Connector specific considerations 
```

Comment:
What do you want to write in this section? be careful, "consideration" is "Prendre en considération" in French